sdk/include/psdk/twain.h

```diff
@@ -1268,7 +1268,7 @@ typedef struct {
 #define TWLG_SWEDISH              TWLG_SWE
 #define TWLG_THAI                 84
 #define TWLG_TURKISH              85
-#define TWLG_UKRANIAN             86
+#define TWLG_UKRAINIAN            86
```

Comment:
Nope! You are breaking compatibility with official headers
https://github.com/twain/twain-specification/blob/master/versions/2.1/twain.h#L1427

Reply:
```suggestion
#define TWLG_UKRANIAN             86
```